screens/components/Fab.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { View, Pressable } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import Icon from 'react-native-vector-icons/Ionicons';
+import { Colors } from './theme';
+
+const Fab = ({ navigation }) => {
```

Comment:
Can we call this something better, like `ActionButton` or `CircleButton`

Reply:
ActionButton will be good I will change it..
This particular FAB is not used this one is the camera one that we removed after discussion.
but there is one more named HomeFab thats being used in HomeScreen.. I will rename that